pkg/api/grpcServer.go

```diff
@@ -78,8 +78,10 @@ func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string,
 		request proto.Message, response proto.Message)) error {
 
 	// tracks attribute state for this stream
-	tracker := s.attrMgr.NewTracker()
-	defer tracker.Done()
+	reqTracker := s.attrMgr.NewTracker()
```

Comment:
Do we ever need/use a singular tracker outside of tests? Should `NewTracker` be updated to ` func NewTracker() (req Tracker, resp Tracker)`?